src/api/index.js

```diff
@@ -404,3 +404,33 @@ export async function determineInstallNamespace() {
     });
   return response;
 }
+
+export function getTriggerTemplates({ namespace } = {}) {
+  const uri = getTektonAPI('triggertemplates', { namespace });
+  return get(uri).then(checkData);
+}
+
+export function getTriggerTemplate({ name, namespace }) {
+  const uri = getTektonAPI('triggertemplates', { name, namespace });
+  return get(uri);
+}
+
+export function getTriggerBindings({ namespace } = {}) {
```

Comment:
shouldn't there be `filters` argument as well?

Reply:
Added, yeah I'll add those next thanks